spec/support/quickbooks_online_helpers.rb

```diff
@@ -2,6 +2,128 @@
 
 module QuickBooksOnlineHelpers
   STUBS = {
+    preferences: {
```

Comment:
Needs to be pluralized.

Reply:
Solved/OK if we rename to `Preferences` plural?